templates/repo/issue/view_content/comments.tmpl

```diff
@@ -279,7 +279,7 @@
 			</span>
 			<div class="detail">
 				<span class="octicon octicon-plus"></span>
-				<span class="text grey"><a href="{{$.RepoLink}}/issues/{{.DependentIssue.Index}}">#{{.DependentIssue.Index}} {{.DependentIssue.Title}}</a></span>
+				<span class="text grey"><a href="{{AppSubUrl}}/{{.DependentIssue.Repo.OwnerName}}/{{.DependentIssue.Repo.Name}}/issues/{{.DependentIssue.Index}}">{{.DependentIssue.Repo.FullName}}#{{.DependentIssue.Index}} - {{.DependentIssue.Title}}</a></span>
```

Comment:
I think that `.DependentIssue` could be `nil` if the user has no permission to read the other repo? Perhaps an `{{if .DependentIssue}}` should be used.

Reply:
Just use `{{.DependentIssue.HTMLURL}}`.

Reply:
updated both, thanks